tools/scenario-player/scenario_player/runner.py

```diff
@@ -143,7 +143,7 @@ def __init__(
             gas_price_strategy=get_gas_price_strategy(settings.get('gas_price', 'fast')),
         )
 
-        self.chain_id = self.client.web3.net.version
+        self.chain_id = int(self.client.web3.net.version)
```

Comment:
Not sure if this breaks stuff

Reply:
shouldn't break anything

Reply:
No, it actually fixes stuff ;)
Ran into this myself in #3604

Reply:
👍